### PR TITLE
removed clock and hard-coded TPS (take 2)

### PR DESCRIPTION
**Pull request by FieldMarshallVague**
changes per the website content changes doc.  basically making TPs values consistent and temporarily removing invalid/broken sections.

### CODE REVIEW

File: explorer/app2/index.html [hunk 1]
```diff
@@ -60,19 +60,13 @@
             <!-- Page 0 -->
             <div id="page-0" class="page-0">
                 <div class="row">
-                    <div class="col has-text-centered">
+                    <!-- <div class="col has-text-centered">
```
Comment (x-wf) [inline explorer/app2/index.html:63]: We can remove this piece of code, if we're not actually using it and it's just commented out. 😃

File: explorer/app2/js/app.js [hunk 2]
```diff
@@ -165,7 +165,8 @@ function updateTickers() {
 
 function updateStats() {
   $('.tps-spot').each(function() {
-    $(this).text(beautifulNumber(getTps()));
+    // $(this).text(beautifulNumber(getTps()));
```
Comment (x-wf) [inline explorer/app2/js/app.js:168]: Same as above.

**Review by x-wf — COMMENTED** [review]
Just a minor suggestions.
Feel free to merge when changed @FieldMarshallVague 👍

File: explorer/app2/index.html (hunk 1 above)
Comment (FieldMarshallVague) [reply explorer/app2/index.html:63]: It's only a temporary commenting-out.  When the tests resume, this code will have to be un-commented.

File: explorer/app2/js/app.js (hunk 2 above)
Comment (FieldMarshallVague) [reply explorer/app2/js/app.js:168]: same as above

**Pull request merged**